torch/ao/quantization/fx/prepare.py

```diff
@@ -560,15 +562,16 @@ def _maybe_insert_input_observer_for_arg_or_kwarg(
     new_arg = arg
 
     is_standalone_module = qhandler is not None and qhandler.is_standalone_module()
-    assert qconfig is not None
     if not is_standalone_module:
+        # Note: qconfig can be None in this branch this we are getting act/fq from
+        # node.meta now
```

Comment:
What happens when `qconfig` is not None and we have an `act/fq` from `node.meta`? Which one will we use?

Reply:
we should just use `act/fq` from `node.meta`, `qconfig` is ignored in this branch